aspnet/fundamentals/localization/sample/Resources/SharedResource.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Dummy class to group shared resources
```

Comment:
Unless I missed it, you might mention in the doc above that you must create a .cs class for any "dummy" resource files.